Client/Frontend/Settings/TPAccessoryInfo.swift

```diff
@@ -49,9 +49,9 @@ class TPAccessoryInfo: ThemedTableViewController {
         topStack.isLayoutMarginsRelativeArrangement = true
 
         sep.backgroundColor = themeManager.currentTheme.colors.borderPrimary
-        sep.snp.makeConstraints { make in
-            make.height.equalTo(0.5)
-            make.width.equalToSuperview()
+        sep.heightAnchor.constraint(equalToConstant: 0.5).isActive = true
+        if let sepSuperView = sep.superview {
+            sep.widthAnchor.constraint(equalTo: sepSuperView.widthAnchor).isActive = true
```

Comment:
This PR includes changes that were done for the other task in https://github.com/mozilla-mobile/firefox-ios/pull/17134 when it shouldn't. Can you remove those (you can revert the changes to that file, and push again on that same branch).

Reply:
Made the changes as per the step above, Kindly review the PR @lmarceau